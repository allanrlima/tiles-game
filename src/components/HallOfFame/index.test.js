import React from "react";
import { shallow } from "enzyme";

import { HallOfFame } from ".";

const props = {
  hideHallOfFame: () => {}
};

describe("HallOfFame", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<HallOfFame {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
