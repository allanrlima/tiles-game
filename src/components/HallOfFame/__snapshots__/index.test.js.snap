// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HallOfFame renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <HallOfFame
    hideHallOfFame={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <ForwardRef>
          Hall Of Fame (: 
           - No Records yet
        </ForwardRef>,
        <ForwardRef />,
        <Button
          onClick={[Function]}
          title="Back to the game"
        />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            "Hall Of Fame (: ",
            " - No Records yet",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Hall Of Fame (: ",
          " - No Records yet",
        ],
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-htpNat",
            "isStatic": true,
            "rules": Array [
              "
  text-transform: uppercase;
  display: flex;
  justify-content: center;
  text-indent: 0.7rem;
  font-weight: 300;
  white-space: nowrap;
  letter-spacing: 0.7rem;
  ",
              "
    @media (max-width: ",
              "48",
              "em) {
      ",
              "
    text-indent: inherit;
    letter-spacing: inherit;
  ",
              ";
    }
  ",
              "
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-htpNat",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [],
        },
        "ref": null,
        "rendered": Array [],
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bxivhb",
            "isStatic": true,
            "rules": Array [
              "
  display: grid;
  grid-template-columns: auto auto;
  grid-template-rows: repeat(5, 72px [col-start]);
  margin-top: 32px;
  ",
              "
    @media (max-width: ",
              "48",
              "em) {
      ",
              "
    font-size: 18px;
    margin-top: 16px;
    display: flex;
    flex-direction: column;
  ",
              ";
    }
  ",
              "
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bxivhb",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "onClick": [Function],
          "title": "Back to the game",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bwzfXH",
        "isStatic": true,
        "rules": Array [
          "
  position: fixed;
  top: 0;
  left: 0;
  display: grid;
  grid-template-rows: 64px auto 64px;
  z-index: 10;
  background-color: #000;
  width: 100%;
  height: 100%;
  font-size: 32px;
  color: #fff;
  padding: 16px;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bwzfXH",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <ForwardRef>
            Hall Of Fame (: 
             - No Records yet
          </ForwardRef>,
          <ForwardRef />,
          <Button
            onClick={[Function]}
            title="Back to the game"
          />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              "Hall Of Fame (: ",
              " - No Records yet",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Hall Of Fame (: ",
            " - No Records yet",
          ],
          "type": Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-htpNat",
              "isStatic": true,
              "rules": Array [
                "
  text-transform: uppercase;
  display: flex;
  justify-content: center;
  text-indent: 0.7rem;
  font-weight: 300;
  white-space: nowrap;
  letter-spacing: 0.7rem;
  ",
                "
    @media (max-width: ",
                "48",
                "em) {
      ",
                "
    text-indent: inherit;
    letter-spacing: inherit;
  ",
                ";
    }
  ",
                "
",
              ],
            },
            "displayName": "styled.div",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-htpNat",
            "target": "div",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          },
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [],
          },
          "ref": null,
          "rendered": Array [],
          "type": Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-bxivhb",
              "isStatic": true,
              "rules": Array [
                "
  display: grid;
  grid-template-columns: auto auto;
  grid-template-rows: repeat(5, 72px [col-start]);
  margin-top: 32px;
  ",
                "
    @media (max-width: ",
                "48",
                "em) {
      ",
                "
    font-size: 18px;
    margin-top: 16px;
    display: flex;
    flex-direction: column;
  ",
                ";
    }
  ",
                "
",
              ],
            },
            "displayName": "styled.div",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-bxivhb",
            "target": "div",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          },
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "onClick": [Function],
            "title": "Back to the game",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bwzfXH",
          "isStatic": true,
          "rules": Array [
            "
  position: fixed;
  top: 0;
  left: 0;
  display: grid;
  grid-template-rows: 64px auto 64px;
  z-index: 10;
  background-color: #000;
  width: 100%;
  height: 100%;
  font-size: 32px;
  color: #fff;
  padding: 16px;
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bwzfXH",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
